src/effect/base.ts

```diff
@@ -92,7 +92,7 @@ export class Base implements BaseObject {
   /**
    * The current time of the target
    */
-  get currentTime (): number {
+  get getCurrentTime (): number {
```

Comment:
You don't need to do this. Since movie.currentTime is always up to date the getter works fine.